plutus-ledger/src/Ledger/Typed/Scripts/MonetaryPolicies.hs

```diff
@@ -31,8 +32,6 @@ import           Ledger.Typed.TypeUtils
 
 type WrappedMintingPolicyType = Data -> Data -> ()
 
--- TODO: in due course when we have monetary policies with redeemers we should add a TypedMintingPolicy interface here
```

Comment:
I still think we should do this!

Reply:
Not in this PR, to be clear, I just think we should leave the TODO. Or maybe just make an issue.

Reply:
Oops